runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java

```diff
@@ -63,4 +63,39 @@ default boolean showOpponentsInMenu()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showVanillaPercentages",
+		name = "Show HP % on the Vanilla HP Bar",
+		description = "Adds a percentage value next to the health value of the vanilla HP Bar",
+		position = 4
+	)
+
+	default boolean showVanillaPercentages()
```

Comment:
Isn't this already covered by `HitpointsDisplayStyle.BOTH`?

Reply:
While I could make it so that the value on the vanilla health bar is equal to that of the opponent information, I thought someone might want just the percentage on the vanilla health bar, and something else on the opponent information panel, that's why this toggle exists